src/include/cart/types.h

```diff
@@ -211,6 +211,7 @@ struct crt_proto_rpc_format {
 	struct crt_req_format	*prf_req_fmt;
 	/** the RPC hander on the server side */
 	crt_rpc_cb_t		 prf_hdlr;
+	crt_rpc_cb_t		 prf_enq;
```

Comment:
I'm working on a patch to enqueue requests in 'cc_rpc_cb', and the request queue is managed by scheduler.